lib/JMAP/Tester/UA/LWP.pm

```diff
@@ -19,6 +19,8 @@ has lwp => (
     my $lwp = LWP::UserAgent->new;
     $lwp->cookie_jar({});
 
+    $lwp->default_header('Content-Type' => 'application/json');
+
```

Comment:
Shouldn't this also be charset utf8

Also, why are we only doing this on the LWP UA and not also the Async UA?